runtime-decompiler/src/main/java/org/jrd/frontend/frame/main/MainFrameView.java

```diff
@@ -134,6 +143,43 @@ public String toString() {
         }
     }
 
+    private void setImageIcon() {
+        class BufferedImageTranscoder extends ImageTranscoder {
+            BufferedImage result;
+
+            @Override
+            public BufferedImage createImage(int width, int height) {
+                return new BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB);
+            }
+
+            @Override
+            public void writeImage(BufferedImage image, TranscoderOutput to) throws TranscoderException {
+                this.result = image;
+            }
+        }
+
+        var transcoder = new BufferedImageTranscoder();
```

Comment:
First usage of `var` and its type inference in the whole project! Congrats :fireworks: